### PR TITLE
RST-1035 ET3 cucumber tests

### PR DESCRIPTION
**Pull request by AlexaVSmith**

### CODE REVIEW

File: features/step_definitions/et3_claiments_details_page_steps.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+end
+
+When(/^I click on next without providing the required claimants details$/) do
+  earnings_and_benefits_page.next
+end
+
+Then(/^I should see the error message saying the claimants details cant be blank$/) do
+  # TODO: behaviour is incorrect
+  expect(claimants_details_page).to have_error_header
+#   expect(claimants_details_page.agree_with_employment_dates_question).to have_error_inclusion
```
Comment (sivvoy) [inline features/step_definitions/et3_claiments_details_page_steps.rb:28]: Can we please remove all unused code?

File: features/step_definitions/et3_respondents_details_page_steps.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  expect(claimants_details_page).to be_displayed
+end
+
+When(/^I click on next without providing the required respondents details$/) do
+  respondents_details_page.next
+end
+
+Then(/^I should see the error message saying the respondents details cant be blank$/) do
+  expect(respondents_details_page).to have_error_header
+  # functionality is wrong for case number
```
Comment (sivvoy) [inline features/step_definitions/et3_respondents_details_page_steps.rb:26]: removed unused code please?

**Review by sivvoy — APPROVED** [review]
apart from commented code I'm 👍

File: features/step_definitions/et3_claiments_details_page_steps.rb (hunk 1 above)
Comment (AlexaVSmith) [reply features/step_definitions/et3_claiments_details_page_steps.rb:28]: As discussed, changes are needed, these relate to https://tools.hmcts.net/jira/browse/RST-1127

File: features/step_definitions/et3_respondents_details_page_steps.rb (hunk 2 above)
Comment (AlexaVSmith) [reply features/step_definitions/et3_respondents_details_page_steps.rb:26]: As discussed, changes are needed, these relate to https://tools.hmcts.net/jira/browse/RST-1127

**Pull request merged**